### PR TITLE
tox: switch to use py3

### PR DESCRIPTION
**Pull request by kshtsk**
Signed-off-by: Kyr Shatskyy <kyrylo.shatskyy@suse.com>

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -6,8 +6,8 @@ RUN apt-get update && apt-get install \
 # Install dependencies:
 COPY requirements.txt .
 ADD . /paddles
-RUN pip install -r requirements.txt
-RUN pip install /paddles/.
+RUN pip3 install -r requirements.txt
```
Comment (tchaikov) [inline Dockerfile:9]:
```
Step 6/11 : RUN pip3 install -r requirements.txt
631 ---> Running in 033b462f61be
632/bin/sh: 1: pip3: not found
633The command '/bin/sh -c pip3 install -r requirements.txt' returned a non-zero code: 127
634The command "docker build . --file Dockerfile --tag ceph-paddles-test" failed and exited with 127 during .
635
636
```

might want to install python3, python3-pip, python3-setuptools and python3-wheel instead.

**Comment by kshtsk** [comment]
@tchaikov jfyi @toabctl will not be going to review in the next two months, he is on long term vacation now

File: Dockerfile (hunk 1 above)
Comment (kshtsk) [reply Dockerfile:9]: Yeah... looking into this

File: paddles/stats.py [hunk 2]
```diff
@@ -18,7 +18,7 @@ def get_client():
     except AttributeError as exc:
         log.info(
             "Could not find statsd configuration; disabling statsd. "
-            "Error message was: %s" % exc.message
+            "Error message was: %s" % str(exc)
```
Comment (tchaikov) [inline paddles/stats.py:21]:
```suggestion
            "Error message was: %s" % exc
```

`log.info()` is able to take care of this.

Comment (kshtsk) [reply paddles/stats.py:21]: yeh

**Comment by kshtsk** [comment]
retest this please

**Comment by kshtsk** [comment]
retest this please

File: paddles/tests/controllers/test_nodes_race.py [hunk 3]
```diff
@@ -1,4 +1,8 @@
-import Queue
+try:
+    from queue import Queue
+except:
+    from Queue import Queue
```
Comment (tchaikov) [inline paddles/tests/controllers/test_nodes_race.py:4]: so we are still compatible with python2.

**Review by tchaikov — APPROVED** [review]

**Pull request merged**